examples/src/main/java/org/apache/spark/examples/streaming/JavaRecoverableNetworkWordCount.java

```diff
@@ -23,8 +23,11 @@
 import java.util.Arrays;
 import java.util.Iterator;
 import java.util.List;
+import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.regex.Pattern;
 
+import org.apache.spark.api.java.function.*;
+import org.apache.spark.streaming.Seconds;
```

Comment:
I generally would not collapse imports to `*`, though the code is inconsistent and people have different feelings. Don't make that change